contributing.md

```diff
@@ -31,7 +31,15 @@ npm install # (if you haven't done so yet)
 npm run build
 ```
 
-This will create an `amp.zip` in the plugin directory which you can install. The contents of this ZIP are also located in the `build` directory which you can `rsync` somewhere as well. 
+This will create an `amp.zip` in the plugin directory which you can install. The contents of this ZIP are also located in the `build` directory which you can `rsync` somewhere as well.
+
+TO create a build of the plugin as it will be deployed to WordPress.org, run:
+
+```bash
+npm run build-release
+```
+
+Note that this will currently take much longer than a regular build because it generates the files required for translation. You also must have WP-CLI installed with the `i18n` package.
```

Comment:
@westonruter You might wanna add a link to https://github.com/wp-cli/i18n-command

Reply:
Regarding the performance: a quick guess is that the command currently scans every folder looking for PHP files to parse. We might wanna exclude this directory (and others) by default. See https://github.com/wp-cli/i18n-command/blob/90f3f8e80a8e7db71e3b1e2d2b26140d5dd9e9fe/src/WordPressCodeExtractor.php#L115-L135 and https://github.com/wp-cli/i18n-command/issues/27.

If you try running `wp i18n make-pot` without a node_modules folder being there, it should definitely be very fast. If not, it's something else.